BioPage.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Peronal Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+    <div class="main">
+        <nav> 
+            <ul>
+                <li><a href="HomePage.html">HOME</a></li>
+                <li><a href="BioPage.html">ABOUT ME</a></li>
+                <li><a href="ExperiencePage.html">EXPERIENCE PAGE</a></li>
+                <li><a href="PortfolioPage.html">PORTFOLIO PAGE</a></li>
+            </ul>
+        </nav>
+        <div class="homeInfo">
+            <h1><span>About Me</span></h1>
+            <p>Born and raised in Georgia (Savannah & Atlanta). I've always had a passion for technlogy.
+                <br>I have been playing video game since I was little, and I always wanted to make one of
```

Comment:
rather than putting line breaks on each line, consider using some styling to justify or left-align this text